spaces/S000176/properties/P000133.md

```diff
@@ -5,6 +5,12 @@ value: false
 refs:
   - mr: MR2048350
     name: General Topology (Willard)
+  - wikipedia: Cut_point
+    name: Cut point on Wikipedia
 ---
 
+Every {P133} space with at least three points has a 
```

Comment:
I think we need to say "Every connected {P133} space ...", since talking about cut points assumes the space is connected to start with (which the plane is).